image_width: 1920
image_height: 1080
camera_name: floating_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [2100.0, 0.0, 960.0,
         0.0, 2100.0, 540.0,
         0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.0, 0.0, 0.0, 0.0, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0,
         0, 1, 0,
         0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [2100.0, 0.0, 960.0, 0.0,
         0.0, 2100.0, 540.0, 0.0,
         0.0, 0.0, 1.0, 0.0]